### PR TITLE
Add support of tblte-common (Note Edge)

### PR DESCRIPTION
**Pull request by micky387**

### CODE REVIEW

**Review by micky387 — COMMENTED** [review]
remove trltecan & trlteusc

**Comment by ripee** [comment]
In line 14 of audio/Android.mk, you forgot to add trltevzw.

Are you sure that trlte should direct to mixer_paths.xml? Won't there be confusion since the unified build works on both trltexx, which needs mixer_paths_eur.xml, and the other variants?

I see your logic, but the non-edge Note 4 isn't like the Edge.

File: audio/Android.mk [hunk 1]
```diff
@@ -9,7 +9,9 @@ LOCAL_MODULE_CLASS := ETC
 LOCAL_SRC_FILES    := mixer_paths_eur.xml
 LOCAL_MODULE_PATH  := $(TARGET_OUT_ETC)
 include $(BUILD_PREBUILT)
-else
+endif
+
+ifeq ($(TARGET_DEVICE),trlte trltetmo trltecan trltespr trlteusc trlteduos)
```
Comment (ripee) [inline audio/Android.mk]: Does trlte need to be here? Will it conflict with trltexx above?

trltevzw definitely needs to be here!

**Comment by micky387** [comment]
it's was i think but _mone say to me (in freenode) to add this.

there is this commit with mixer_patch for trlte https://github.com/samsung-apq8084/android_device_samsung_trlte/blob/cm-14.1/releasetools/mixer_paths_eur.sh

**Comment by ripee** [comment]
You mean to move that to trlte-common? As long as it works, but "trlte" is not itself a variant designation, so should it be under mixer_paths.xml? I may be wrong about trlte, but please add trltevzw.

**Comment by micky387** [comment]
_mone say: "in this case trlte is 'trltexx trltedt trltetmo trltevzw trlte...' I've kept those conditions for users that want to build for a single device (not the unified "trlte"). Building for trlte has to use mixer_path.xml and if that build is being flashed to trltexx/dt it will use mixer_path_eur.xml thanks to this: https://github.com/samsung-apq8084/android_device_samsung_trlte/blob/cm-14.1/releasetools/mixer_paths_eur.sh"

**Review by micky387 — COMMENTED** [review]
add trltevzw

**Comment by ripee** [comment]
Ok, I was just making sure about trlte. Thank you for adding trltevzw.

**Comment by micky387** [comment]
no problem.
we can talk in freenode if you want

**Comment by underscoremone** [comment]
@micky387, did you try building for tblte with those changes? What about blobs? Can you use the trlte-comon ones? If yes we have to regen the mk files in proprietary_vendor_samsnug/trlte-common, if not we have to play around with the extract-scripts and probably add more ifeq here: https://github.com/samsung-apq8084/android_device_samsung_trlte-common/blob/cm-14.1/trlte.mk#L21

**Comment by micky387** [comment]
yes,all working great.
call,data,audio. All work fine.
By curiousity,i'm try to use trlte-common blob.
All working without error (normal,it's same devices) no need other thing. i'm test 2/3 days and if it's work great ,i 'll delete tblte-common blobs.
So if for you all is all right,i can merge this commits ...?

**Comment by ripee** [comment]
I'll run a trlte build with micky387's trlte-common fork and make sure that in-call audio isn't lost with his changes, which it shouldn't be but I'd like to make sure.

I can confirm that tblte compiled properly and it's in the tblte folder in my BasketBuild. trlte build with micky's trlte-common fork is running now.

**Comment by micky387** [comment]
ok Keep me informed

**Comment by ripee** [comment]
I can now confirm that in-call audio, speakers and mics, work properly on my trltexx phone running trlte built with micky387's trlte-common fork with this commit, so I see no problem in merging this commit 👍 

On a related matter, I can build lineage-14.1 for tblte daily for you if you'd like, as I see you're working on Oreo.

**Comment by micky387** [comment]
thanks ripee. 
Yes working on oreo.
I will push in 1st time, all commits for fix build error.
in second time, treble for apq8084-common needed for all apq devices.

_mone,you are ok too for merge ?

**Pull request merged**